packages/widget-chat/src/components/title-bar/index.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+import styles from './styles.css';
+
+import Avatar from '../avatar';
+
+class TitleBar extends Component {
+  shouldComponentUpdate(nextProps) {
+    return this.props.user !== nextProps.user;
+  }
+
+  render() {
+    const {user} = this.props;
+    const {displayName} = this.props.user;
+    return (
+      <div className={classNames(`title-bar`, styles.titleBar)}>
+        <Avatar user={user} />
+        <h2 className={classNames(`title`, styles.title)}>{displayName}</h2>
+      </div>
+    );
+  }
+}
+
+TitleBar.propTypes = {
+  user: PropTypes.object.isRequired
```

Comment:
Are we sure we want to be pulling in the full user? Since it's a dumb component, just pass the displayName and the avatar?